pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -181,17 +166,16 @@ func parseKeys(packedKeys string) (private, public interface{}) {
 func encodeVCToJSON(vcBytes []byte) {
 	credential, err := verifiable.NewCredential(vcBytes, verifiable.WithNoCustomSchemaCheck())
 	if err != nil {
-		log.Println(fmt.Errorf("failed to decode credential: %w", err))
-		abort()
+		abort("failed to decode credential: %v", err)
 	}
 	encoded, err := credential.MarshalJSON()
 	if err != nil {
-		log.Println(fmt.Errorf("failed to encode credential: %w", err))
-		abort()
+		abort("failed to encode credential: %v", err)
 	}
 	fmt.Println(string(encoded))
 }
 
-func abort() {
+func abort(msg string, args ...interface{}) {
```

Comment:
is really necessary to have `abort()`

I believe the above functions should be Test functions..

Reply:
If I'm not mistaken this _is_ in a test file, seems like the right place for test helpers.

Reply:
Wait nvm, I get what you're saying and agree - `Test_` functions which can use `require` instead of abort, and get rid of abort.